GovUk.Frontend.AspNetCore.Extensions/Styles/_tpr-notification-banner.scss

```diff
@@ -0,0 +1,40 @@
+﻿@import '_variables.scss';
+@import 'govuk/base';
+
+$tpr-notification-banner__heading-line-height: 1.5em;
+$tpr-notification-banner__content-line-height: 1.5em;
+$tpr-notification-banner-font-size: 1.4em;
+
+.tpr-notification-banner{
+    background-color: $tpr-colour-eastern-blue;
+    padding: govuk-spacing(6);
+    margin-bottom: govuk-spacing(8);
+}
+.tpr-notificaiton-banner__content {
+    text-align: center;
+    color: $tpr-colour-white;
+
+    > .govuk-body {
+        color: $tpr-colour-white;
+        font-size: $tpr-notification-banner-font-size;
+        line-height: $tpr-notification-banner__content-line-height;
+        margin: 0 0 govuk-spacing(4) 0;
+    }
+
+    > .govuk-button-group {
+        justify-content: center;
+        margin: 0;
+    }
+
+    > .govuk-button{
+        margin: 0;
+    }
+}
+
+.tpr-notification-banner__heading {
+    font-family: $govuk-font-family;
+    font-weight: $tpr-font-weight-bold;
+    line-height: $tpr-notification-banner__heading-line-height;
+    font-size: $tpr-notification-banner-font-size;
+    margin: 0 0 govuk-spacing(2) 0;
+}
```

Comment:
There are significant contrast issues that violate WCAG 1.4.11 https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html 
> "For active controls any visual information provided that is necessary for a user to identify that a control is present and how to operate it must have a minimum 3:1 contrast ratio with the adjacent colors"

I haven't run that contrast check so you would need to (or Jodi, because you need to work with her on this) but to my eye the following look like they would fail.

The default state of the default button and cancel link, and the outline of the secondary button:

![Default state of buttons](https://user-images.githubusercontent.com/1260550/215524131-1735659a-f6a3-4571-9901-efc85542ce1d.png)

The hover state of the default button:

![Buttons showing default button in hover state](https://user-images.githubusercontent.com/1260550/215524417-fd2e185a-80c9-42e0-adf9-0bcad591fbda.png)

The hover state of the secondary button:

![Buttons showing secondary button in hover state](https://user-images.githubusercontent.com/1260550/215524530-7cbfe510-20f1-4fd2-99a6-a9fb03886667.png)

The hover state of the cancel link:

![Buttons showing cancel link in hover state](https://user-images.githubusercontent.com/1260550/215524633-b2f542c9-2fd0-4019-a140-4cbdc695bb1a.png)

This also applies to links included in the text of the notification banner, which are fine on the GOV.UK implementation because they are on a white background.

Reply:
Jodi has said that only a reversed button and a link would be inside a notification banner. I have added the styles required for a reversed button and added the configuration in Umbraco so it can be made a reversed button with the radio buttons like warning and secondary buttons. I used this [prototype](https://xd.adobe.com/view/5ba7aba5-bf2b-431b-7084-513a0af3106a-2d96/screen/62ef7c21-8187-42c3-818b-2e09dada8feb/specs/?hints=off).

Jodi also said the hover state does not fail WCAG 1.4.11 as non-text contrast is not required to reach the 3:1 contrast ratio with the background.

Reply:
There is still a problem with the unvisited link style in the success banner:
![success banner showing unvisited link](https://user-images.githubusercontent.com/1260550/218807219-6e59602f-cd36-4b1e-ab5e-ba9f48d2fdb2.png)

Tab to the dismiss button on the blue banner. Press and hold space and you will see the focus + active state. which has insufficient text contrast:

![focus plus active state of button](https://user-images.githubusercontent.com/1260550/218808940-36593994-7289-402f-890d-da373440800f.png)

Reply:
The text contrast for unvisited links and the focus + active button state has been resolved.

I have raised the hover contrast issue with Jodi again however she still disagrees. Also when demoing the reversed button in the notification banner, Jodi did not raise any concerns. Maybe this is something to pick up again later?

Reply:
I've discussed with Jodi and reviewed more WCAG guidance and I now agree that the design passes.